tests/components/plex/test_config_flow.py

```diff
@@ -474,12 +500,14 @@ async def test_callback_view(hass, aiohttp_client):
         DOMAIN, context={"source": "user"}
     )
     assert result["type"] == "form"
-    assert result["step_id"] == "start_website_auth"
+    assert result["step_id"] == "user"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)